doc/tutorial/pinball-tutorial.rst

```diff
@@ -391,34 +391,68 @@ And when it's built and run:
     24	High score: 100
     25	Locking paddles.
 
-Default Handlers
-================
+Timers
+------
 
 Now that we have users, they've started doing bad and unexpected
 things to our pinball machine. Someone figured out they could lift the
 front of the machine to prevent the ball from dropping and get cheaty
 high scores.
 
-Any-Event
----------
-
 Fortunately for us, there's a tilt sensor in our pinball machine. All
 we have to do is hook up the event to our Smudge machine.
 
 .. code-block:: c
    :include: 05_pinball.smudge
    :linenos:
 
-Combined with implementations of ``displayError`` and ``startTimer``
-in **05_main.c**, this new *lockout* state ignores all events until
-the timer expires.
+This uses the enter and exit events we just talked about to guarantee
+that the timer is limited in scope to the ``lockout`` state. All
+that's left is to implement the timers.
 
-The way it accomplishes this is with an *any-event* handler. The
-*any-event* is a special event. It's spelled with a single ``_`` and
-catches all of the events not explicitly listed in the state. This
-way, we can ignore all of the player's events for a little while to
-give them some time to think about whether or not cheating at pinball
-is a winning strategy for life.
+Fortunately for us, Smear has support for cancellable timers. To use
```

Comment:
Super nit: two paragraphs in a row start with "Fortunately for us".

Reply:
Well caught. This bit involved heavy use of scissors and I didn't proofread it as well as I should have.